templates/releases.hbs

```diff
@@ -0,0 +1,78 @@
+{{> header}}
+
+{{#if varsb.show_search_form}}
+<div class="container landing">
+  <h1 class="brand"><i class="fa fa-cubes fa-fw"></i> Docs.rs</h1>
+
+  <form action="/releases/search" method="GET" class="landing-search-form">
+    <div><input class="search-input" name="query" type="text" placeholder="Search"></div>
+    <div class="buttons">
+      <button type="submit" class="pure-button pure-button-normal">Search</button>
+      <button type="submit" class="pure-button pure-button-normal" id="i-am-feeling-lucky-button">I'm Feeling Lucky</button>
+    </div>
+  </form>
+  <script type="text/javascript" charset="utf-8">
```

Comment:
My snippet addon is adding charset, tbh I have no idea, but sure I'll remove :-)